permissions/plugin-blueocean-executor-info.yml

```diff
@@ -0,0 +1,10 @@
+---
+name: "blueocean-executor-info"
+paths:
+- "org/jenkins-ci/plugins/blueocean-executor-info"
+developers:
+- "jamesdumay"
+- "michaelneale"
+- "vivek"
+- "kzantow"
+---
```

Comment:
I wonder whether this document separator messes things up…

Reply:
@daniel-beck how do I know if it does? (I copy paste)

Reply:
Looks good.